Correct SA findings alerts API (#6179)
opensearch-project/documentation-website: _security-analytics/api-tools/alert-finding-api.md

Parameter | Description 
:--- | :---
`detector_id` | The ID of the detector used to fetch alerts. Optional when the `detectorType` is specified. Otherwise required.

> `detectorType` | The type of detector used to fetch alerts. Optional when the `detectorId` is specified. Otherwise required.

Reviewer: ```suggestion
`detectorType` | The type of detector used to fetch alerts. Optional when the `detector_Id` is specified. Otherwise required.
```

`severityLevel` | Used to filter by alert severity level. Optional.

> `alertState` | Used to filter by alert state. Possible values: ACTIVE, ACKNOWLEDGED, COMPLETED, ERROR, DELETED. Optional.

Reviewer: ```suggestion
`alertState` | Used to filter by alert state. Possible values are ACTIVE, ACKNOWLEDGED, COMPLETED, ERROR, or DELETED. Optional.
```

`sortString` | This field specifies which string Security Analytics uses to sort the alerts. Optional.

> `sortOrder` | The order used to sort the list of findings, either `asc` or `desc`. Optional.

Reviewer: ```suggestion
`sortOrder` | The order used to sort the list of findings. Possible values are `asc` or `desc`. Optional.
```

`DELETED` | Someone deleted the detector or trigger associated with this alert while the alert was ongoing.

---

## Acknowledge alerts

> Gives an acknowledgement when an alert is triggered.

Reviewer 2: "Provides" or "Sends" instead of "Gives"?

Author: ```suggestion
Sends an acknowledgement when an alert is triggered.
```

```

---

## Get findings

The Get findings API returns findings based on the detector attributes.

### Parameters

You can specify the following parameters when getting findings.

Parameter | Description 
:--- | :---
`detector_id` | The ID of the detector used to fetch alerts. Optional when the `detectorType` is specified. Otherwise required.

> `detectorType` | The type of detector used to fetch alerts. Optional when the `detectorId` is specified. Otherwise required.

Reviewer: ```suggestion
`detectorType` | The type of detector used to fetch alerts. Optional when the `detector_id` is specified. Otherwise required.
```

> `sortOrder` | The order used to sort the list of findings, either `asc` or `desc`. Optional.

Reviewer: ```suggestion
`sortOrder` | The order used to sort the list of findings. Possible values are `asc` or `desc`. Optional.
```
